src/Tribe/Service_Providers/Context.php

```diff
@@ -304,7 +334,8 @@ static function ( $post_type ) {
 					'read' => [
 						Tribe__Context::FUNC          => [
 							static function () {
-								return ! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
+								return ! empty( tribe_get_request_var( Venue::POSTTYPE,
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 87ba508582abaf63d61bf1e1548c7475</sub>